README.md

```diff
@@ -130,6 +130,7 @@ isDayHighlighted: PropTypes.func,
 // internationalization props
 displayFormat: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
 monthFormat: PropTypes.string,
+weekDayFormat: PropTypes.string,
```

Comment:
Is there a moment proptype we could use for "valid format"?

Reply:
Technically I think any string is a valid format (you can include arbitrary strings by surrounding them with brackets)